src/frontend/custom-sequencer.ts

```diff
@@ -0,0 +1,29 @@
+const Sequencer = require("@jest/test-sequencer").default;
+
+/** A sequencer that ensures that ensures that the slowest tests
+ * all land on different shards. */
+class CustomSequencer extends Sequencer {
+  shard(tests, { shardIndex, shardCount }) {
```

Comment:
not typed, I guess no need to?

Reply:
it's actually a pain to set up because it gets picked up in the actual app build so we need to ts-ignore it. Will need you help eventually to split the tsconfig...